### ISSUE TITLE
Feature Request： Local responder

### ISSUE THREAD

**Issue by linonetwo** [kookma/Solution#1]
Sometimes I want a solution to open a local tiddler, because I store some of my friends' name as tiddler.

So I hope this will open a local tiddler in the current browser tab, while currently it will open a new tab.

<img width="287" alt="截屏2020-08-05 上午11 04 42" src="https://user-images.githubusercontent.com/3746270/89367397-78d41b00-d70b-11ea-90a9-a8a6439b84a9.png">

**Comment by kookma** [comment]
@linonetwo Do you mean the link appears on a solution tiddler can open a local tiddler instead of a url? If so, I look and see how can I add this!

**Comment by linonetwo** [comment]
Yes, I think if url is a hash fragment, just remove the `target="_blank"` from the `a` tag?

**Comment by kookma** [comment]
Have you tried that, if so you may submit a PR. If not let me see how to fix.

**Comment by linonetwo** [comment]
I don't know how to express this in tw template:

```jsx
<a href={{!!url}} target={url.startWith('#') ? "_blank" : null}>
  <$list filter="[enlist<responders>join[, ]]" variable=item><$text text=<<item>> /></$list>
</a>
```

I have written too much React ...

**Comment by linonetwo** [comment]
`<a href={{!!url}} target={{[url]+regexp[^#]else{{url}} }}>`, oh it seems wrong...

**Comment by kookma** [comment]
No problem, I will fix it!

**Comment by kookma** [comment]
Fixed in Solution 1.4.0: If the URL field contains a tiddler name (even one with spaces), a link to the local tiddler is now created. However, the link still opens in a new tab or window to avoid losing focus.
